packages/block-editor/src/layouts/constrained.js

```diff
@@ -146,8 +147,25 @@ export default {
 			</>
 		);
 	},
-	toolBarControls: function DefaultLayoutToolbarControls() {
-		return null;
+	toolBarControls: function DefaultLayoutToolbarControls( {
+		layout = {},
+		onChange,
+		layoutBlockSupport,
+	} ) {
+		const { allowSwitching, allowJustification = true } =
+			layoutBlockSupport;
+
+		if ( allowSwitching || ! allowJustification ) {
```

Comment:
Any reason we're checking for `allowSwitching` here? It's pretty much a legacy attribute, because the controls it exposes when enabled haven't been maintained since they were added. No harm in leaving it here though.

Reply:
My guess is that the pull request was created based on the code here:

https://github.com/WordPress/gutenberg/blob/16420c4ec6c6868a10d08e46b08612cd726e776b/packages/block-editor/src/layouts/flex.js#L98-L100

@shreya0204 Can you refactor it as follows?

```javascript
const { allowJustification = true } = layoutBlockSupport;
if ( ! allowJustification ) {
	return null;
}
```

Reply:
@t-hamano Certainly! I've refactored the code as requested. Let me know if it needs any further adjustments. Thanks!